python/cugraph/components/cc_wrapper.pyx

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from c_cc cimport *
+#from libcpp cimport bool
+from c_graph cimport *
+#from libcpp cimport cugraph_cc_t
+from libc.stdint cimport uintptr_t
+import cudf
+#from pygdf import Column
+import numpy as np
+
+cpdef weak_cc(G, connect_type = CUGRAPH_WEAK):
+    """
+    Find the distances and predecessors for a breadth first traversal of a graph.
+    
+    Parameters
+    ----------
+    G : cugraph.graph
+        cuGraph graph descriptor, should contain the connectivity information as an
+        adjacency list.
+    connect_type : cugraph_cc_t
+        Weak (CUGRAPH_WEAK), or Strong (CUGRAPH_STRONG) (not implemented, yet)
+    
+    Returns
+    -------
+    df : cudf.DataFrame
+        df['labels'][i] gives the label id of the i'th vertex
+       
+        
+    Examples
+    --------
+    >>> M = read_mtx_file(graph_file)
+    >>> sources = cudf.Series(M.row)
+    >>> destinations = cudf.Series(M.col)
+    >>> G = cuGraph.Graph()
+    >>> G.add_edge_list(sources,destinations,none)
+    >>> dist, pred = cuGraph.weak_cc(G, CUGRAPH_WEAK)
```

Comment:
I guess dist and pred are misnomers, here, and according to the description above, this function returns a single cudf.DataFrame.

Reply:
I fixed that.